coalib/bearlib/languages/definitions/TypeScript.py

```diff
@@ -0,0 +1,26 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class TypeScript:
+    aliases = 'ts'
+    extensions = '.ts', '.tsx'
+    comment_delimiter = '//'
+    multiline_comment_delimiters = {'/*': '*/'}
+    string_delimiters = {'"': '"', "'": "'"}
+    multiline_string_delimiters = {}
```

Comment:
https://stackoverflow.com/questions/35225399/multiline-string-with-typescript-angular2 tells me typescript has multiline string support with backticks?

Reply:
Yes, it does.

Reply:
Ok will add them later.